firefox-ios/Client/Application/RemoteSettings/Application Services/ASSearchEngineUtilities.swift

```diff
@@ -46,11 +46,13 @@ struct ASSearchEngineUtilities {
             return URLQueryItem(name: $0.name, value: value)
         }
         // From API docs: "This may be skipped if `{searchTerm}` is included in the base."
-        if let searchArg = searchURL.searchTermParamName, !searchURL.base.contains("{searchTerm}") {
-            // Note: term vs terms is not a typo.
+        // Note: there is a typo in the docs, the value is searchTerms (plural).
```

Comment:
nit: should we reference the docs? not sure where the typo was. Or we can add a JIRA ticket number here with more context. 

Also interesting to see that the previous comment says:
`// Note: term vs terms is not a typo.` Do you have context on this 🤔 or seems like this was a mistake?